.github/workflows/openstack/reboot_watch

```diff
@@ -10,21 +10,40 @@ use POSIX;
 my $RETVAL  = 1;
 my $RETRIES = 0;
 
+# Verify REGEX isn't already in the log before we go into tail mode.  The logodump here is sgnificantly faster than File::Tail.
+
+open LOGFILE, ELEVATE_LOG_PATH;
+my @log = <LOGFILE>;
+close LOGFILE;
+
+# Jump if it's found.
+
+if ( grep { /${REGEX}/m } @log ) {
```

Comment:
You sure you don't wanna `/\Q${REGEX}\E/m` that? Might make sense so long as you aren't passing in regexp escapes/patterns but just strings to grep for.

Reply:
Making a change for this one.